src/util/dsp/acovb.hpp

```diff
@@ -7,8 +7,8 @@ using namespace gam;
 
 
 void acovb(gsl::span<float> data){
-    const unsigned N = data.size();
-    const unsigned M = 2*(N -1); // FFT order
+    const int N = (int) data.size();
+    const int M = 2*(N -1); // FFT order
 
     // FFT is in-place and we need more space, so we first copy the data to a buffer, and zero-pad
     float buf[M];
```

Comment:
this wont work, you're not allocating any memory.
We dont actually have a good solution to this right now, as there is no way to allocate two audio buffers which are guarantied to be consecutive, or one double sized one. You're gonna have to do it using a vector in the class for now.